gcloud_utils/ml_engine.py

```diff
@@ -5,6 +5,8 @@
 import logging
 import re
 import time
+import pickle
+import tensorflow as tf
```

Comment:
I don't think we should use tensorflow as dependency... We're going to support other frameworks like PyTorch... it's not a good approach